webapp/src/js/enketo/widgets/rdtoolkit-capture.js

```diff
@@ -0,0 +1,203 @@
+        </div>
+        <div>
+          <span class="rdt-label">
+            ${window.CHTCore.Translate.instant('report.rdtoolkit_capture.rdtoolkit_preview_time_read')} 
+          </span>
+          <span class="rdt-value">${capturedTest.timeRead}</span>
+        </div>
+        <div>
+          <span class="rdt-label">
+            ${window.CHTCore.Translate.instant('report.rdtoolkit_capture.rdtoolkit_preview_image')} 
+          </span>
+          <img src="data:image/png;base64,${capturedTest.image}">
+        </div>
+        <br>
+        <div>
+           ${window.CHTCore.Translate.instant('report.rdtoolkit_capture.rdtoolkit_preview_next_action')} 
+        </div>
+      `);
+  }
+
+  function updateFields($widget, capturedTest) {
+    setFieldValue($widget, 'rdtoolkit_session_id', capturedTest.sessionId);
+    setFieldValue($widget, 'rdtoolkit_results', JSON.stringify(capturedTest.results));
+    setFieldValue($widget, 'rdtoolkit_results_description', capturedTest.resultsDescription);
+    setFieldValue($widget, 'rdtoolkit_time_read', capturedTest.timeRead);
+    setFieldValue($widget, 'rdtoolkit_state', capturedTest.state);
+    setFieldValue($widget, 'rdtoolkit_time_started', capturedTest.timeStarted);
+    setFieldValue($widget, 'rdtoolkit_time_resolved', capturedTest.timeResolved);
+    setFieldValue($widget, 'rdtoolkit_image', capturedTest.image);
+  }
+
+  function getDate(dateTime) {
+    return dateTime && moment(dateTime).isValid() ? moment(dateTime).format('LLL'): '';
+  }
+
+  function getForm() {
+    return window.CHTCore.Enketo.getCurrentForm();
+  }
+
+  function getFieldValue(form, fieldName) {
+    if (!form || !fieldName) {
+      return;
+    }
+
+    return form.model.$
+      .find(fieldName)
+      .text();
+  }
+
+  function getFormattedResult(results) {
+    if (!results) {
+      return '';
+    }
+
+    let description = '';
+
+    results.forEach(item => {
+      const test = window.CHTCore.Translate.instant(item.test);
+      const result = window.CHTCore.Translate.instant(item.result);
+
+      description += `${description ? ', ' : ''}${test || item.test}: ${result || item.result}`;
+    });
+
+    return description;
+  }
+
+  function setFieldValue($widget, fieldName, value) {
+    if (!fieldName || value === undefined) {
+      return;
+    }
+
+    $widget
+      .find(`input[name$=${fieldName}]`)
+      .val(value)
+      .trigger('change');
+  }
+
+  $.fn[pluginName] = function(options, event) {
+    return this.each(function() {
+      const $this = $(this);
+      let data = $this.data(pluginName);
+
+      options = options || {};
+
+      if (!data && typeof options === 'object') {
+        $this.data(pluginName, (data = new Rdtoolkitcapturewidget(this, options, event)));
+
+      } else if (data && typeof options === 'string') {
+        data[options](this);
+      }
+    });
+  };
+
+  module.exports = {
+    'name': pluginName,
+    'selector': '.or-appearance-rdtoolkit_capture',
```

Comment:
The part after the `or-appearance-` is visible to the app builder since it is the appearance they use in the XLSForm, and luckily/conveniently the convention for those appearances is to use dashes (and coincides with [this comment](https://github.com/medic/cht-core/pull/7093#issuecomment-837544860)).

Reply:
I've replaced the underscore with dashes, except for field names where we use underscore to keep consistency, example: [patient_id](https://github.com/medic/cht-core/blob/8c1151f812819a54d24073d38856591a51e50ac3/config/default/forms/app/pnc_danger_sign_follow_up_mother.xml#L164) or __patient_uuid